sdkhelpers.go

```diff
@@ -12,30 +12,28 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package metric
+package otel
 
 import (
 	"context"
-
-	"go.opentelemetry.io/otel/api/core"
 )
 
-// InstrumentImpl is the implementation-level interface Set/Add/Record
-// individual metrics without precomputed labels.
-type InstrumentImpl interface {
+// Instrument interface for implementations of individual metrics without
+// precomputed labels.
+type Instrument interface {
 	// AcquireHandle creates a Handle to record metrics with
 	// precomputed labels.
-	AcquireHandle(labels LabelSet) HandleImpl
+	AcquireHandle(labels LabelSet) Handle
 
 	// RecordOne allows the SDK to observe a single metric event.
-	RecordOne(ctx context.Context, number core.Number, labels LabelSet)
+	RecordOne(ctx context.Context, number Number, labels LabelSet)
 }
 
-// HandleImpl is the implementation-level interface to Set/Add/Record
-// individual metrics with precomputed labels.
-type HandleImpl interface {
+// Handle interface for implementations of individual metrics with precomputed
+// labels.
+type Handle interface {
```

Comment:
This is confusing, what does `otel.Handle` means?  Maybe change to `MetricsHandle`?